content/en/docs/ambient/extensibility/wasm-module-distribution/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Distributing WebAssembly Modules
+description: Describes how to make remote WebAssembly modules available for ambient mode.
+weight: 1
+keywords: [extensibility,Wasm,WebAssembly,Ambient]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
```

Comment:
could be a follow up PR, would be really nice to have a test for this.

Reply:
Yes, I'll include the test as a follow up PR